client/src/components/flashcard/answerScale.js

```diff
@@ -0,0 +1,65 @@
+import { Button, Card, Typography } from "@material-tailwind/react";
+import React, { useEffect, useState } from "react";
+import { useDispatch } from "react-redux";
+import { refreshFlashCardAsync } from "../../store/flashcards/thunks";
+
+const Scale = (props) => {
+  const confidenceColors = ["red", "orange", "amber", "green", "light-green"];
+
+  const dispatch = useDispatch();
+
+  const handleRefresh = (quality) => {
+    dispatch(
+      refreshFlashCardAsync({
+        quality: quality,
+        cardId: props.cardId,
+      })
+    );
+  };
+
+  return (
+    <Card className="w-full bg-transparent flex flex-row justify-evenly shadow-none">
+      {confidenceColors.map((color, i) => (
+        <div key={i} className="flex flex-col items-center">
+          <Button
+            disabled={props.disabled}
+            variant="gradient"
+            color={color}
+            className="rounded-full w-10 h-10 p-0"
+            onClick={() => {
```

Comment:
It might be better if for button handlers you can pull out the code into a helper function. Might make it clearer what the code is doing

Reply:
That's a great suggestion, but since it's not a major change, I'll take note of this for now and will refactor the code before submitting a subsequent pr.